router/worker.go

```diff
@@ -1012,6 +1016,16 @@ func (w *worker) postStatusOnResponseQ(respStatusCode int, payload json.RawMessa
 		}
 		return
 	}
+	switch errorAt {
```

Comment:
Can we add additional check of status code and set only if it's failed. just as a safe check. even though this function gets returned if it's success before this block